_source/_posts/2022-10-13-oktane-for-developers.md

```diff
@@ -0,0 +1,80 @@
+﻿---
+layout: blog_post
+title: "Hello Developers, Time to Get Your Oktane On"
+author: havi-hoffman
+by: advocate
+communities: [security] 
+description: "Oktane 2022 happens in San Francisco from November 8-10 and we'd love to meet you there at the Developer Hub. We've designed presentations and workshops with developers in mind." 
+tags: [announcement, conference] 
+tweets: 
+- "In person or live online, Oktane 22 happens November 8-10 in San Francisco, and you're invited. 
+🧿 Come for the content 🧿 Stay for the conversation 🧿 Leave with useful learnings! #Oktane22 "
+image: blog/oktane22/oktane-online.jpg
+type: awareness
+---
+
+There are plenty of great reasons for developers to join us in San Francisco next month for Oktane 22, November 8-10, 2022. We're back together live and in person—celebrating our 10th annual identity event—and we've built a developer experience at the conference with folks like you in mind. 
+
+{% include toc.md %}
+
+## Why come to Oktane 
+
+Developers make it possible for anyone to use Okta's identity cloud to connect securely to any online technology, anywhere. Whether you support Okta in the workplace, or build on our Auth0 by Okta offering for authorizaton and authentication, whether you're a newcomer or a veteran of the IAM (Identity and Access Management) space, we'd love to meet you at Oktane. Engineers and developer advocates will share roadmaps and knowledge through talks, workshops, and 1:1 conversations. Come for the learning and stay for the good times. As always, we want to hear from you about your pain points, feature requests, and ideas for the future. 
+
+## Developer workshops and presentations
+
+Our speakers are developers from Okta and Auth0, as well as engineers from around the world who work with our platform. They'll cover best practices for protecting your front-end and back-end with OAuth. You'll learn how Okta APIs and SDKs can help take care of authentication, authorization, and user management for your application. We're bringing our latest hands-on workshops and challenges (road-tested and refined at Developer Days), as well as 1:1 consultations with solutions architects to the Oktane Developer Hub. 
```

Comment:
@havi Megan Donegan has updated the agenda so all internal speakers work for Okta. We should probably update this too.

Reply:
I updated to "Auth0 by Okta" in the body copy, per guidance from Cliff